firstvoices/backend/views/search/utils.py

```diff
@@ -0,0 +1,45 @@
+from backend.models.dictionary import DictionaryEntry
+from backend.search_indices.dictionary_entry_document import (
+    ELASTICSEARCH_DICTIONARY_ENTRY_INDEX,
+)
+
+
+def hydrate_objects(raw_objects):
```

Comment:
Is raw_objects a technical term? Could we call them whatever elasticsearch calls them? Hits or search_results or whatever?